lambdas/choice/test/test_writer.py

```diff
@@ -1,5 +1,134 @@
-from src.writer import handler
+import boto3
+import os
+import pytest
+from src.writer import write_to_dynamo
```

Comment:
It seems to me like this is testing implementation, not behavior. What we really care about is what happens when the handler is called, right? Are you doing this so you can inject the DynamoDB client? That shouldn't be necessary with stubbing.

Reply:
Yeah, the import of the write_to_dynamo method here was to allow injection of a stubbed Dynamo client. I understand your concern about testing implementation vs. behavior, but can you point me toward whichever way you prefer to handle stub injection?

Reply:
I might be making assumptions re: stubbing based on my experience with Node, but I _think_ you shouldn't need to inject the stubs:
```python
from src.writer import handler, dynamodb
from botocore.stub import Stubber

def test_function():
    with Stubber(dynamodb.meta.client) as stub:
        # Test things!
        pass
```

Reply:
Nope, you're totally right. Importing the dynamodb variable was the solution. Thanks for the tip!